### ISSUE TITLE
More options from ggplot for plotAbundanceDensity

### ISSUE THREAD

**Issue by artur-sannikov** [microbiome/miaViz#122]
`plotAbundanceDensity` function does not accepts arguments from ggplot, for example `jitter` layout does not have `width` option and uses `geom_point()` as its basis but it does not accept[ geom_point](https://ggplot2.tidyverse.org/reference/geom_point.html) options.

**Comment by antagomir** [comment]
This issue should be moved to miaViz?

**Comment by artur-sannikov** [comment]
You are right. It's possible to [transfer issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository) if you have write access.

**Comment by TuomasBorman** [comment]
Another thing is that it might be simper to modify the ggplot object that comes from the plotAbundanceDensity function (instead of adding new parameters). Can you check that option? If that is easier, then we could modify examples

**Comment by antagomir** [comment]
> You are right. It's possible to [transfer issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository) if you have write access.

@artur-sannikov You are in miaverse-developers team and that has write access to miaViz. Does it work?

**Comment by antagomir** [comment]
One limitation with ggplot is that changing the geom_* afterwards is not so handy (e.g from violin to boxplot). It is possible and perhaps it would be wise to first create & show an example of that and then think if that is too complex, justifying shortcuts.

**Comment by TuomasBorman** [comment]
@artur-sannikov

- Can you describe what was the idea of supporting more options (why we need support for these ggplot options)?
- Can you create an example on how to get the desired behavior without modifying miaViz?

--> The we can decide if we add the support

**Comment by artur-sannikov** [comment]
I think what I wanted to achieve is to have `width` and `height` parameters from this plot: https://ggplot2.tidyverse.org/reference/geom_jitter.html

```R
library(tidyverse)
data(mpg)

ggplot(mpg, aes(x = cyl, y = hwy)) +
  geom_jitter(width=0.5, height=0.3)
```

Some arguments are set [here](https://github.com/microbiome/miaViz/blob/devel/R/plotAbundanceDensity.R#L309).

**Comment by antagomir** [comment]
See also 
#142 
#107 
#90

**Comment by antagomir** [comment]
Any volunteers to pick up this one (including the linked issues)?
